orchestra/todos/serializers.py

```diff
@@ -110,7 +113,8 @@ def create(self, validated_data):
         return instance
 
     def update(self, instance, validated_data):
-        validated_data = self._set_step_to_validated_data(validated_data)
+        if validated_data.get('step') is not None:
```

Comment:
Just curious: when will step be None here?

Reply:
When we are partially updating todos.